loopchain/blockchain/blockchain.py

```diff
@@ -261,6 +261,12 @@ def __add_block(self, block: Block, vote: Vote = None):
                     'block_height': self.__block_height
                 }})
 
+            # stop leader complain timer
```

Comment:
It makes coupling between blockchain and consensus algorithm implementations. I cannot help it now, but I think it would be better to remove it later.